.github/workflows/publish.yaml

```diff
@@ -0,0 +1,12 @@
+# Run tests and publish to all registries if successful.
+name: Publish
+on:
+  release:
+    types: [published]
```

Comment:
Updated the release type trigger to `published` as that appears to be the current recommendation.